### ISSUE TITLE
Screen reader is not announcing the expanded/collapsed state for the dropdown controls present in the header section in Windows.

### ISSUE THREAD

**Issue by msftedad** [wixtoolset/issues#9027]
### WiX Version

5.0.0

### .NET or MSBuild or Visual Studio Version

.NET 6.0

### HeatWave Version

1.0.3

### Windows Version

Win24H2

### Repro Repo

https://www.wix.com/blog

### Repro Steps

1. Turn on Narrator.
2. Open URL:  https://www.wix.com/blog in the latest edge browser.
3. Press tab key from top of the page to move to the dropdown controls like Categories and resources present in the header section.
4. Verify whether the screen reader is announcing the expanded/collapsed state for the Categories and resources controls or not.

### Actual Result

Screen reader is not announcing the expanded/collapsed state for the dropdown controls present in the header section in Windows.

https://github.com/user-attachments/assets/a89ba02c-288b-43ab-8560-f51741d4aaa6

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/6e4b6f3f-2f70-429f-a987-d1e50980214c" />

### Expected Result

The screen reader should announce the expanded or collapsed state of dropdown controls in the header section on Windows. When a dropdown is expanded or collapsed. It should announce it as for ex: Categories Expanded'.
Ensure that the dropdown should close using ESC key and the screen reader should announce as collapsed once the dropdown is closed.

### Acknowledgements

- [x] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be sent to Discussions).

**Comment by barnson** [comment]
The WiX Toolset is not wix.com.
